app/javascript/components/WithTooltip.tsx

```diff
@@ -1,28 +1,56 @@
-import cx from "classnames";
 import * as React from "react";
 
+import { classNames } from "$app/utils/classNames";
```

Comment:
I originally used Radix's tooltip here, but it required a bunch of changes and broke in subtle ways, so [removed](https://github.com/antiwork/gumroad/pull/1968/commits/0f5062325ffff3fbbd6a3f6bd82d87b651048231) it. Don't think it's worth it; ideally we eventually get rid of tooltips entirely as they're bad UX (we did the same thing in Flexile)